.github/workflows/windows-tests.yml

```diff
@@ -0,0 +1,132 @@
+name: "Windows Tests"
+
+on:
+  pull_request:
+    branches:
+      - "v*.*"
+      - "master"
+      - "feature/*"
+  push:
+    tags:
+      - "*"
+    branches:
+      - "v*.*"
+      - "master"
+      - "feature/*"
+
+jobs:
+  build:
+    name: "Build Windows DLLs"
+    runs-on: windows-2022
```

Comment:
Is there any way to enforce that this is standardized between both jobs?

[jobs.<job_id>.runs-on](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idruns-on) suggests that this can be a variable.

Not sure this is needed since we only have two jobs, but wanted to ask anyway.

Reply:
Per the [Workflow syntax documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions), the only other place where to put such values is `env`. However, values set there can't be arrays, so even though we could reference an environment variable in the matrix definition, we can't set the list of PHP versions in an environment variable to reuse it in both matrixes.